### ISSUE TITLE
Using Camaleon with existing database and models

### ISSUE THREAD

**Issue by Tashows** [owen2345/camaleon-cms#567]
Hi,

I found out Camaleon yesterday and was really impressed by the admin interface. Is there any way to integrate the admin interface with an app I have already built?

Going through the process of creating custom content groups and fields, when I already have everything setup and configured, is a big kill. On top of that I don't really need Camaleon to access everything on my app. I just need to provide cms access for articles and a custom contest model. Till now I was using Active Admin, which has a way for you to configure which models you want to register and use on its admin interface. Is there a way to do something similar with Camaleon?

I went through the documentation, but it gets confusing after the installation part. Plus the installation shows how to setup a new app. What about using existing models, migrations, controllers, services? I can't figure out how to get Camaleon to work with my database and existing files.

What you've built looks pure awesome. I am not sure if it's supposed to have the capabilities I described but if it does, then it's a game-changer!

Thanks,
Tasos

**Comment by owen2345** [comment]
Hi @Tashows 
I created another installation manual here:
http://camaleon.tuzitio.com/documentation/category/40757-developer-docs/installation-existent-project.html
Sorry for the delay, I was in a vacation.
Regards,

**Comment by Tashows** [comment]
Hi @owen2345 

After looking through the docs some more, I think what I want to do might actually be possible, but the documentation (not just the one you gave in your previous response, but in general) is very hard to follow. I might give this another chance, once proper documentation is available or once I have more free time to figure this out.

Thanks for your time

**Comment by hopewise** [comment]
I tried to install it on my existing app, but it did not generate any new migration files
I did:
    RAILS_ENV=development rake db:migrate:status VERBOSE=true

all is up and Camaleon migrations are not listed.

I tried:

    RAILS_ENV=development rake db:migrate VERBOSE=true

nothing happened.. I thought its ok, but when I run the app I got error:

> PG::UndefinedTable: ERROR: relation "cama_term_taxonomy" does not exist LINE 5: WHERE a.attrelid = '"cama_term_taxonomy"'::re... ^ : SELECT a.attname, format_type(a.atttypid, a.atttypmod), pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod FROM pg_attribute a LEFT JOIN pg_attrdef d ON a.attrelid = d.adrelid AND a.attnum = d.adnum WHERE a.attrelid = '"cama_term_taxonomy"'::regclass AND a.attnum > 0 AND NOT a.attisdropped ORDER BY a.attnum

How can I do migrations properly ? I just followed common steps but no luck..
